CONTRIBUTING.md

```diff
@@ -0,0 +1,200 @@
+# Contributing
+
+## Where to start
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+The best place to start is to check the [issues](https://github.com/rust-practice/cargo-leet)
```

Comment:
I was wondering if we should also point them at the discussion in case they find something in there more interesting.